app/controllers/items_controller.rb

```diff
@@ -0,0 +1,39 @@
+class ItemsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  def index
+    @items = Item.all
+  end
+  def new
+    @categories = Category.all
+    @subcategories = SubCategory.all
```

Comment:
@sub_categories

Reply:
@sulemanuzair we need to handle category and subcateogory for item in a separate task

Reply:
ok